src/theory/quantifiers/sygus/sygus_grammar_cons.cpp

```diff
@@ -1294,6 +1370,28 @@ void CegGrammarConstructor::mkSygusDefaultGrammar(
         sdtBool.addConstructor(kind, cargs);
       }
     }
+    else if (types[i].isFloatingPoint())
+    {
+      Trace("sygus-grammar-def") << "...add FP predicates" << std::endl;
+      std::vector<Kind> fp_unary_predicates = {FLOATINGPOINT_ISN,
+                                               FLOATINGPOINT_ISSN,
+                                               FLOATINGPOINT_ISZ,
+                                               FLOATINGPOINT_ISINF,
+                                               FLOATINGPOINT_ISNAN,
+                                               FLOATINGPOINT_ISNEG,
+                                               FLOATINGPOINT_ISPOS};
+      for (const Kind kind : fp_unary_predicates)
+      {
+        sdtBool.addConstructor(kind, cargs);
+      }
+      std::vector<Kind> fp_binary_predicates = {FLOATINGPOINT_LEQ,
+                                                FLOATINGPOINT_LT};
```

Comment:
I note that IEEE equality isn't included.  This is probably not a bad thing.  As long as it is a intentional.

Reply:
Right, since we rewrite it anyways. I also intentionally excluded `min` and `max` in order to not introduce UFs.